packages/opentelemetry-resources/src/detectors/BrowserDetector.ts

```diff
@@ -61,5 +62,18 @@ class BrowserDetector implements Detector {
   }
 }
 
+//Add Browser related attributes to resources
+export function getBrowserAttributes():ResourceAttributes {
+  const browserAttribs:ResourceAttributes={};
```

Comment:
Not sure why linter is not complaining about this. Would you mind adding the necessary spacing between the words?

```suggestion
  const browserAttribs : ResourceAttributes = {};
```

Also other